web/client/reducers/dimension.js

```diff
@@ -58,6 +59,13 @@ export default (state = {}, action) => {
     case RESET_CONTROLS: {
         return set('data', undefined, set('currentTime', undefined, set('offsetTime', undefined, state)));
     }
+    case MAP_CONFIG_LOADED: {
+        const config = action?.config || {};
+        return {...state,
+            currentTime: config?.dimensionData?.currentTime,
+            offsetTime: config?.dimensionData?.offsetTime
+        };
+    }
```

Comment:
I should keep the epic in the middle if possible, or we can not control or avoid this operation with a flag. 
This is technically an operation that the timeline plugin does to initialize. Not a standard behavior of the dimensions state.